python/lsst/meas/extensions/psfex/psfex.py

```diff
@@ -307,6 +307,7 @@ def showPsf(psf, set, ext=None, wcsData=None, trim=0, nspot=5,
 def getFlags(tab=None):
     flagKeys = [
         "base_PixelFlags_flag_edge",
+        "base_PixelFlags_flag_nodata",
```

Comment:
Is this where you had in mind @arunkannawadi?

Reply:
Yes, exactly.